src/client/eppo-client.spec.ts

```diff
@@ -28,7 +28,11 @@ export async function init(configurationStore: IConfigurationStore<Flag | Obfusc
     sdkVersion: '1.0.0',
   });
   const httpClient = new FetchHttpClient(apiEndpoints, 1000);
-  const configurationRequestor = new FlagConfigurationRequestor(configurationStore, httpClient);
+  const configurationRequestor = new ConfigurationRequestor(
+    httpClient,
+    configurationStore,
+    undefined,
+  );
```

Comment:
Regarranged and expanded the signature of the `ConfigurationRequestor`'s constructor. It is now: 
```ts
constructor(httpClient,flagConfigurationStore, banditConfigurationStore)
```
So I pass in `undefined` for all these non-bandit tests.